docs/contributor_guide.rst

```diff
@@ -83,43 +127,44 @@ When creating a Taichi program using ``ti.init(arch=desired_arch, **kwargs)``, p
 Testing
 -------
 
-Tests should be added to ``taichi/tests/python``.
-
-Use ``ti test`` to run all the tests. Use ``ti test_verbose`` to test with verbose outputs.
+Tests should be added to ``taichi/tests``.
 
+Use ``ti test`` to run all the tests.
+Use ``ti test -h`` to see more testing options.
```

Comment:
Also `ti test -v` for verbose, `ti test <name(s)>` for specific tests.

Reply:
Cool! Added to the doc.